internal/cmd/local/command_kafka_start.go

```diff
@@ -25,20 +24,28 @@ import (
 )
 
 var (
-	statusBlacklist = []string{"Pulling fs layer", "Waiting", "Downloading", "Download complete", "Verifying Checksum", "Extracting", "Pull complete"}
+	statusBlacklist = []string{"Pulling fs layer", "Waiting", "Download complete", "Verifying Checksum", "Pull complete"}
 )
 
-type imagePullOut struct {
-	Status string `json:"status"`
+type ImagePullResponse struct {
+	Status   string `json:"status"`
+	Error    string `json:"error,omitempty"`
+	Progress string `json:"progress,omitempty"`
+	ID       string `json:"id,omitempty"`
 }
 
 func (c *Command) newKafkaStartCommand() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:   "start",
 		Short: "Start a single-node instance of Apache Kafka.",
 		Args:  cobra.NoArgs,
 		RunE:  c.kafkaStart,
 	}
+
+	cmd.Flags().String("kafka-rest-port", "", "The port number for kafka REST. If not specified, a random free port will be used.")
```

Comment:
```suggestion
	cmd.Flags().String("kafka-rest-port", "", "The port number for Kafka REST. If not specified, a random free port will be used.")
```

Reply:
Shouldn't we default to 8082, and then a random free port? I think most users are used to Kafka REST being on that port. https://docs.confluent.io/platform/current/kafka-rest/index.html#deployment

Reply:
sounds good!